opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -50,8 +50,10 @@
 #include <opm/core/utility/parameters/ParameterGroup.hpp>
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableManager.hpp>
-
 #include <opm/common/data/SimulationDataContainer.hpp>
+
+#include <dune/common/timer.hh>
```

Comment:
ok. the reason why I opted for `Dune::Timer` instead of `Opm::StopWatch` was that `opm-core` is scheduled to be dissolved?